### PR TITLE
feat: get historical rates from voltz api

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: src/services/v1-indexer/getHistoricalRatesGCloud.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+export const getFixedRatesGCloud = async (
+  chainId: SupportedChainId,
+  vammAddress: string,
+  startTimestamp: number,
+  endTimestamp: number,
+): Promise<HistoricalRate[]> => {
+  try {
+    const baseUrl = getServiceUrl('fixed-rates');
+    const url = `${baseUrl}/${chainId}/${vammAddress}/${startTimestamp}/${endTimestamp}`;
+    const res = await axios({
```
Comment (ssbarbee) [inline src/services/v1-indexer/getHistoricalRatesGCloud.ts:20]: you can transform this into axios.get<HistoricalRate[]>(url, {       withCredentials: false }); to add type safety of the res.data.

also improves readability imho.

File: src/services/v1-indexer/getHistoricalRatesGCloud.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+export const getVariableRatesGCloud = async (
+  chainId: SupportedChainId,
+  rateOracleAddress: string,
+  startTimestamp: number,
+  endTimestamp: number,
+): Promise<HistoricalRate[]> => {
+  try {
+    const baseUrl = getServiceUrl('variable-rates');
+    const url = `${baseUrl}/${chainId}/${rateOracleAddress}/${startTimestamp}/${endTimestamp}`;
+    const res = await axios({
```
Comment (ssbarbee) [inline src/services/v1-indexer/getHistoricalRatesGCloud.ts:44]: see comment: https://github.com/Voltz-Protocol/v1-sdk/pull/294/files#r1197704439

Comment (avniculae) [reply src/services/v1-indexer/getHistoricalRatesGCloud.ts]: [ab441d4](https://github.com/Voltz-Protocol/v1-sdk/pull/294/commits/ab441d4d20c43630a1af18c0ca119dc476ca7676)

File: src/services/v1-indexer/getHistoricalRatesGCloud.ts (hunk 1 above)
Comment (avniculae) [reply src/services/v1-indexer/getHistoricalRatesGCloud.ts]: [ab441d4](https://github.com/Voltz-Protocol/v1-sdk/pull/294/commits/ab441d4d20c43630a1af18c0ca119dc476ca7676)

**Pull request merged**